core/blockchain.go

```diff
@@ -498,25 +494,16 @@ func SysCallContract(contract common.Address, data []byte, chainConfig params.Ch
 	}
 	blockContext := NewEVMBlockContext(header, GetHashFn(header, nil), engine, author)
 	evm := vm.NewEVM(blockContext, txContext, ibs, &chainConfig, vmConfig)
-	if isBor {
-		ret, _, err := evm.Call(
-			vm.AccountRef(msg.From()),
-			*msg.To(),
-			msg.Data(),
-			msg.Gas(),
-			msg.Value(),
-			false,
-		)
-		if err != nil {
-			return nil, nil
```

Comment:
I think this was the reason why `isBor` was handled separately, to not propagate the error upwards